contrib/direnv

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+# Look for the .config folder in the current directory and up
+find_prj_config() (
+	local old_pwd
+	while [[ $old_pwd != $PWD ]]; do
+		if [[ -d .config ]]; then
```

Comment:
Yes, this can be too ambiguous. And I already sort of thought about this problem, but then chose not to address it on the first iteration. But it needs to be addressed before merge.

Reply:
addressed in 44d2b5fcb558d9864c290459ad096dba13aaa025.

@zimbatm is that imlementation good enough for our purpose here?